Qt6.5.3(?)+6.6.0-rc emits new warnings and causes tests to be
aborted even if did not really fail. Should be fixed[1] in 6.6.0
final and cleanable later.

[1] https://bugreports.qt.io/browse/QTBUG-117200
--- a/pytest.ini
+++ b/pytest.ini
@@ -61,2 +61,4 @@
     ^QBackingStore::endPaint\(\) called with active painter; did you forget to destroy it or call QPainter::end\(\) on it\?$
+    # ...and Qt 6.5.3+
+    ^QItemSelectionModel: Selecting when no model has been set will result in a no-op\.
 xfail_strict = true
